### ISSUE TITLE
Will Whipper give up if 5th rip attempt is successful?

### ISSUE THREAD

**Issue by the-confessor** [whipper-team/whipper#449]
I haven't run into this condition yet, but I noticed this looking at the code.

It appears that if the first 4 attempts to rip a track fail, then the 5th attempt succeeds, Whipper would abort the operation anyway.

From cd.py:

```python
                if tries == MAX_TRIES:
                    logger.critical('giving up on track %d after %d times',
                                    number, tries)
                    raise RuntimeError(
                        "track can't be ripped. "
                        "Rip attempts number is equal to 'MAX_TRIES'")
```

Should the condition actually be tries >= MAX_TRIES?

**Comment by ABCbum** [comment]
Yeah, i think just move the block
```
if tries = MAX_TRIES:
  ......
```
in to the while loop before it should solve the problem since if the rip attempt is successful then it will break the loop right away or maybe just start with `tries = 1` then move `tries += 1` to the end of each loop. Right ?

**Comment by the-confessor** [comment]
Yes something like that would also work. As long as it doesn't give up when the most recent attempt to rip the track is successful.

**Comment by JoeLametta** [comment]
@the-confessor Thanks for reporting this issue.
The bug should have been fixed in commit caa2c8b27d6d70fe444b6ac98254df128ab2fb9e.
